nifi-nar-bundles/nifi-standard-services/nifi-hbase_2-client-service-bundle/nifi-hbase_2-client-service/src/main/java/org/apache/nifi/hbase/HBase_2_ClientService.java

```diff
@@ -413,8 +448,20 @@ public void shutdown() {
         if (connection != null) {
             try {
                 connection.close();
-            } catch (final IOException ioe) {
-                getLogger().warn("Failed to close connection to HBase due to {}", new Object[]{ioe});
+            } catch (final Exception e) {
+                getLogger().warn("Failed to close connection to HBase due to {}", new Object[]{e});
```

Comment:
Recommend rewording the message and passing the exception argument so that the stack trace will be included.
```suggestion
                getLogger().warn("HBase connection close failed", e);
```